scripts/build.ps1

```diff
@@ -224,7 +236,12 @@ $BaseBuildDir = Join-Path $RootDir "build"
 $ArtifactsDir = Join-Path $BaseArtifactsDir "bin" $Platform
 $BuildDir = Join-Path $BaseBuildDir $Platform
 
-$ArtifactsDir = Join-Path $ArtifactsDir "$($Arch)_$($Config)_$($Tls)"
+if ("" -eq $ExtraArtifactDir) {
+    $ArtifactsDir = Join-Path $ArtifactsDir "$($Arch)_$($Config)_$($Tls)"
+} else {
+    $ArtifactsDir = Join-Path $ArtifactsDir "$($Arch)_$($Config)_$($Tls)_$($ExtraArtifactDir)"
+}
+
```

Comment:
Theoretically, a lot of tools (e.g. test.ps1) would need to be updated as well, right?

Reply:
If we wanted to run the builds, yes. I can do that either here or in a separate PR.

Reply:
We should add a test for the system libcrypto variant of openssl on Linux. Please add that, and only update the scripts that need to be updated for that to happen.